bundles/org.eclipse.ui.themes/css/light/e4-light_tabstyle.css

```diff
@@ -30,6 +30,7 @@
 	padding: 0px;
 	color: '#org-eclipse-ui-workbench-INACTIVE_TAB_TEXT_COLOR';
 	swt-draw-custom-tab-content-background: true;
+	swt-unselected-hot-tab-color-background:'#org-eclipse-ui-workbench-INACTIVE_TAB_BG_START';
```

Comment:
```suggestion
	swt-unselected-hot-tab-color-background:'#org-eclipse-ui-workbench-ACTIVE_TAB_BG_START';
```

Reply:
Thanks for finding out. I took over your proposal. Can you pls. test again?